sdk/plugin/grpc_system.go

```diff
@@ -18,6 +18,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var errMissingSystemView = errors.New("missing system view implementation: this method should not be called during plugin setup")
```

Comment:
nit: Can we provide a bit more of a hint about when they _can_ call the method? I think the `Initialize` function is the first time they should reliably be able to call into the system and storage views.

Reply:
:+1: